js/components/controlBar.js

```diff
@@ -219,6 +221,56 @@ var ControlBar = React.createClass({
     this.props.controller.state.focusedControl = null;
   },
 
+  /**
+   * Will handle the keydown event when the controlBar is active and it will restrict
+   * tab navigation to elements that are within it when the player is in fullscreen mode.
+   * @private
+   * @param {Object} evt Keydown event object.
+   */
+  handleControlBarKeyDown: function(evt) {
+    if (
+      evt.key !== CONSTANTS.KEY_VALUES.TAB
+      || !this.props.controller.state.fullscreen
```

Comment:
Not super important, I think we usually put the && or || at the end of the line .

Reply:
Ok, I can update this